local-geth.sh

```
@@ -0,0 +1,126 @@
+--http \
+--http.corsdomain='*' \
+--http.vhosts='*' \
+--http.addr=0.0.0.0 \
+--http.port=8545 \
+--http.api=web3,eth,debug,txpool,net,admin,personal \
+--ws \
+--ws.addr=0.0.0.0 \
+--ws.port=8546 \
+--ws.origins='*' \
+--ws.api=web3,eth,debug,txpool,net,admin,personal \
+--networkid=$CHAIN_ID \
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)